src/legacy/core_plugins/vis_type_timelion/public/helpers/timelion_request_handler.ts

```diff
@@ -33,9 +31,26 @@ interface Stats {
   sheetTime: number;
 }
 
-interface Sheet {
-  list: Array<Record<string, unknown>>;
-  render: Record<string, unknown>;
+export interface Series {
+  _global?: boolean;
+  _hide?: boolean;
+  _id?: number;
+  _title?: string;
+  color?: string;
+  data: Array<Record<number, number>>;
+  fit: string;
+  label: string;
+  split: string;
+  stack?: boolean;
+  type: string;
+}
+
+export interface Sheet {
+  list: Series[];
+  render: {
```

Comment:
Looks like this type isn't correct - this is what the API returns:
<img width="400" alt="Screenshot 2020-01-15 at 10 56 18" src="https://user-images.githubusercontent.com/1508364/72424150-d0b62f00-3785-11ea-805e-97cef5d0237c.png">

Reply:
But according to this code https://github.com/elastic/kibana/blob/41601ff913b1be5c7c6e0c2d96ed0935c8c94227/src/legacy/core_plugins/timelion/public/directives/chart/chart.js#L39 the API can return `render` too, or it will be set by default. Yeah, `render` definitely should be an optional property.

Reply:
Okay, after resolving this https://github.com/elastic/kibana/pull/52069#discussion_r366785414 we don't need `type` from `render`, but `grid` property is still used https://github.com/elastic/kibana/blob/41601ff913b1be5c7c6e0c2d96ed0935c8c94227/src/legacy/core_plugins/timelion/public/panels/timechart/schema.ts#L84